npu.user.js

```diff
@@ -1319,9 +1319,9 @@ var npu = {
 
 					window.setTimeout(function() {
 						npu.colourSignedExams();
-					}, 250);
+					}, 1000);
 				}
-			}, 100);
+			}, 500);

```

Comment:
Neptun server response times on BME is a lot higher than on others like ELTE or Unideb. I'm talking about ~800ms versus 40-80 ones.

In the long run, the script might as well just be made sentient that it listens to the reponse times and saves it in its database and uses timeout intervals accordingly.